docs/man_pages/index.md

```diff
@@ -41,6 +41,7 @@ Command | Description
 [mobileframework](project/configuration/mobileframework.html) | Lists all supported versions of the current development framework.
 [mobileframework&nbsp;set](project/configuration/mobileframework-set.html) | Sets the selected development framework version for the project.
 [update-kendoui](lib-management/update-kendoui.html) | Updates or adds Kendo UI Core or Kendo UI Professional to your project.
+[resource](resource.html) | Lists information about splashscreens and icons for Android, iOS and WP8 or creates icons or splashscreens for each platform from a single high-resolution image.
```

Comment:
Lists information about the image resources for all mobile platforms.

(Note that WP8 is not true for NativeScript projects)

Add a line for
```
[resource create](resource-create.html) | Creates image resources for all mobile platforms from a single high-resolution image.
```

Apply these changes throughout all applicable "Related Commands" sections as well.